WEB/Src/PerformanceCollector/Perf.Tests/QuickPulse/QuickPulseServiceClientTests.cs

```diff
@@ -1216,9 +1247,11 @@ public void QuickPulseServiceClientSubmitsInstanceNameToServiceWithSubmitSamples
             // ARRANGE
```

Comment:
QuickPulseServiceClientSubmitsInstanceNameToServiceWithSubmitSamples now checks both instance and role, let's rename the test to reflect its new purpose.